### PR TITLE
Adding solution for 2012

### LINKED ISSUE #1709: Leetcode Question Number 2012
Leetcode Question Number 2012 solution needs to be added in the repo . 
 The link for the question is [this](https://leetcode.com/problems/sum-of-beauty-in-the-array).
 Accepted Languages C++ / JAVA . 
 You Only need to add the  class Solution in the file. 
 Make Sure you have read the contributions guidelines before creating an PR.

### PR DESCRIPTION
**Pull request by moulikagrawal**
Fixes #1709 

My code sample is in : 
- [x] CPP
- [ ] Java 

Screenshot of working code sample  : 

image

## Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My code sample is formatted and works correctly 

[Contributor Guide]: https://github.com/amanv8060/Leetcode-Questions/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (amanv8060) [inline .vscode/settings.json:1]: delete this file

**Review by amanv8060 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (moulikagrawal) [reply .vscode/settings.json]: deleted this sir @amanv8060 , kindly check

**Comment by amanv8060** [comment]
Format check is failing, please format the file

**Comment by moulikagrawal** [comment]
done @amanv8060  sir

**Comment by amanv8060** [comment]
LGTM , thanks for the contribution . (Star the repo if it helped you ).

**Pull request merged**